students/HarveyHerela/lesson05/mailroom3.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+
+donors = [{"name": ("Chase", "Zuma"), "donations": [25 * x for x in range(2)]},
+          {"name": ("Marshall", "Rocky"), "donations": [30 * x for x in range(2, 4)]},
+          {"name": ("Skye", "Rubble"), "donations": [45 * x for x in range(10, 13)]}]
+
+
+def get_name(prompt):
+    """Anything can be a name, so accept anything as a name."""
+    getting_name = True
+
+    while getting_name:
+
+        print("Type 'list' for a list of names in the database, '~' to go back")
+        response = input(prompt)
+        response = response.strip()
+
+        if response.lower() == "list":
+            for d in donors:
+                print("{0} {1}".format(*d["name"]))
+        else:
+            getting_name = False
+    return response
+
+
+def send_thank_you():
+    """Prompts for a donor name and donation amount, then prints a 'thank you' message to the screen."""
+    firstname = get_name("First name: ")
+
+    if firstname == "~":
+        return
+
+    lastname = get_name("Last name: ")
+
+    if lastname == "~":
+        return
+
+    amount = "nada"
+    while not amount.isnumeric():
+        print("Donation amount, or '~' to go to main menu:")
+        amount = input("$ ")
+        amount = amount.strip()
+        if amount == '~':
+            return
+    amount = float(amount)
+
+    # Add the donation
+    name = (firstname, lastname)
+    for d in donors:
+        if name in d:
```

Comment:
This should be `if name in d.values():` because the `in` operator checks for the existence of keys in the dict